lib/network/connection_manager.go

```diff
@@ -117,6 +117,13 @@ func (c *ConnectionManager) AllValidators() []string {
 //   the number of validators which are currently connected
 //
 func (c *ConnectionManager) CountConnected() int {
+	c.RLock()
+	defer c.RUnlock()
+	return c.countConnectedUnlocked()
+}
+
+// Ditto
```

Comment:
Please remove this comment, I think the method name explains well.

Reply:
Done